selenium_tests/basic_test.py

```diff
@@ -32,11 +32,13 @@ def test_zero_price_purchase(self):
 
         # Click the Enroll Now button on dashboard
         self.wait().click(lambda driver: driver.find_element_by_class_name("enroll-button"))
+        self.wait().click(lambda driver: driver.find_element_by_class_name("pay-button"))
```

Comment:
oh oops, this comment was supposed to be in response to the next one! :laughing: